pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
BaSH is very picky about what is executable. Try (note the **dot + forward slash**):

    $ ./ModuleMismatchSimulator.py

works for me.

Also on a *nix system, the executable bit would also have to be flipped. Since you are on Mac OSX can you do this?

    $ chmod 755 ModuleMismatchSimulator.py

this will make it executable by everyone

type `ls -l ModuleMismatchSimulator.py` to see _all_ the file attributes.